src/backend/distributed/replication/multi_logical_replication.c

```diff
@@ -1808,7 +1808,7 @@ TotalRelationSizeForSubscription(MultiConnection *connection, char *command)
 	{
 		char *resultString = PQgetvalue(result, 0, 0);
 
-		remoteTotalSize = pg_strtouint64(resultString, NULL, 10);
+		remoteTotalSize = SafeStringToUint64(resultString, NULL, 10);
```

Comment:
@JelteF do you know why we only have `*64*`versions of these functions? Why not *32* exists? For those, we seem to be able to use `pg_strtoint32`

Reply:
Mostly because they were not needed at the time. These functions were added because `atol` was on the banned APIs list. I didn't know about the pg_strtoXXX functions back then. Also, I haven't looked close enough at these custom pg_strtoXXX functions to see if they catch the same errors as our custom function.

Reply:
I don't think pgXX functions do the rigorous checks that SafeString does. But, I doubt if the checks are really needed. So, for now are you fine with using pg_XXX32 versions?